### ISSUE TITLE
is this project dead?

### ISSUE THREAD

**Issue by simonckenyon** [ReKotlin/rekotlin-router#27]
i'm using it in a production app.
there are bugs.
i've asked some questions.
no answer after a month.
should i be using something else?

**Comment by kmmraj** [comment]
No, this project is very much alive. We missed your issues - will get back on that soon.
